tests/configlet/util/configlet.py

```diff
@@ -58,8 +59,12 @@ def get_port_channel():
         pc_intf["{}|{}/31".format(pc_name, tor_data["ip"]["local"])] = {}
     if tor_data["ipv6"]["local"]:
         pc_intf["{}|{}/126".format(pc_name, tor_data["ipv6"]["local"])] = {}
-    ret.append({ "PORTCHANNEL_INTERFACE": pc_intf })
+    if pc_intf:
+        if is_version_2019_higher():
+            pc_intf["{}".format(pc_name)] = {}
```

Comment:
Doesn't this equals to pc_intf[pc_name] = {}? Why doing the format?

Reply:
Good catch and agree.